### DISCUSSION TITLE
Configure Fail2ban for customized services

### DISCUSSION THREAD

**Discussion by hack3rcon** [fail2ban/fail2ban#3696] [Q&A] [answered]
Hello,
**GitLab** uses Nginx and PostgreSQL. The address of the Nginx log file in GitLab is `/var/log/gitlab/nginx/`. Is the following configuration file correct?
```
[nginx-http-auth]
enabled = true
port     = http,https
logpath = /var/log/gitlab/nginx/*error.log
findtime = 600
bantime = 7200
maxretry = 3
```
Thank you.

**Comment by sebres** [comment, marked as answer]
See https://github.com/fail2ban/fail2ban/discussions/3688#discussioncomment-8679688
